mlprimitives/jsons/keras.Sequential.LSTMTimeSeriesRegressor.json

```diff
@@ -67,16 +76,19 @@
             "metrics": {
                 "type": "list",
                 "default": [
-                    "accuracy"
+                    "mse"
                 ]
             },
             "layers": {
                 "type": "list",
                 "default": [
                     {
-                        "class": "keras.layers.Input",
+                        "class": "keras.layers.LSTM",
                         "parameters": {
-                            "shape": "input_shape"
+                            "input_dim": "input_dim",
+                            "input_length": "input_length",
+                            "units": "lstm_hidden_units",
```

Comment:
Please call this `lstm_units`, following the pattern `{layer_name}_{parameter_name}`